sql/topology/nodeNetwork.sql

```diff
@@ -44,8 +44,8 @@ DECLARE
   /*
    * Author: Nicolas Ribot, 2013 ; Adrien Berchet, 2020
   */
-  edge_table TEXT := $1;
-  tolerance TEXT := $2;
+--   edge_table TEXT := $1;
```

Comment:
remove the comment of these 2 lines
it is the workarround to give name to the compulsory parameters